west.yml

```diff
@@ -63,7 +63,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 4c996ecd07e576524a98405b2af6c01c56cb83e5
+      revision: a7b0003fac963c2e9f9f0edbc9fabc552d7981b4
```

Comment:
While https://github.com/nrfconnect/sdk-zephyr/pull/1913 is sill not merged
`pull/1913/head` is used (unless something changed in recent weeks?)